sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/udf/GetFeedRangeForPartitionKeyValues.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark.udf
+
+import com.azure.cosmos.implementation.SparkBridgeImplementationInternal
+import com.azure.cosmos.spark.CosmosPredicates.requireNotNullOrEmpty
+import org.apache.spark.sql.api.java.UDF2
+
+
+@SerialVersionUID(1L)
+class GetFeedRangeForPartitionKeyValues extends UDF2[String, String, String] {
+  override def call
+  (
+    partitionKeyDefinitionJson: String,
+    partitionKeyValue: String
```

Comment:
and also for the partitionKeyValue, should we constraint into only String type? because by theory, it can be String, Boolean, Number etc